src/styles/_catalog.scss

```diff
@@ -13,10 +12,23 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+
+    box-sizing: border-box;
   }
 
   &__card:not(:nth-child(4n)) {
```

Comment:
i think this is redundant, please check and delate.

Reply:
yep, I believe you can remove it